environment.yaml

```diff
@@ -0,0 +1,9 @@
+name: dask-saturn
+channels:
+  - defaults
+dependencies:
+  - python=3.7
+  - pip
+  - pip:
+      - requests
+      - distributed
```

Comment:
Prefer to conda install these as well. And pin distributed to match what's in the dockerfile

Reply:
what dockerfile?

Reply:
What dockerfile indeed. I was thinking about the one in https://github.com/saturncloud/images/tree/master/daskdev-sat but that is both unrelated and doesn't pin distributed. So ignore that comment...